### ISSUE TITLE
Whenever someone press the submit button in contact page , an email should be sent to my mail

### ISSUE THREAD

**Issue by nitin-pandita** [nitin-pandita/Nitin-Fashionwear#21]
Fixed #IssueNumber

Check (put an 'x' between the square brackets) everything which applies:

- [ ] I have added the issue number for which this pull request is created.

@nitin-pandita, please let me know about this update.

**Comment by nitin-pandita** [comment]
required mail is mr.nitinpandita@gmail.com

**Comment by naim16031988** [comment]
do you mean IF someone tries to log in or sign in  in the contact you should receive an email 
what information you wanna get in the email?

**Comment by nitin-pandita** [comment]
> do you mean IF someone tries to log in or sign in in the contact you should receive an email what information you wanna get in the email?

yes if someone login or sign up, I should get the mail about the username and password

**Comment by naim16031988** [comment]
ok .I can do that,  i Know how to do it with JS 
but, In order for you to receive emails you should sign in to emailJS and link your account Gmail to it .

**Comment by nitin-pandita** [comment]
> Contributor

@naim16031988 I have signed in with my mail mr.nitinpandita@gmail.com

**Comment by naim16031988** [comment]
I dont know how to do it with just e-mail adresse . The way i wanna do it is : 

First, you need to create an account with EmailJS

Then creat a new service and link your gmail account .
Create a template with your service and configure the email form you want to receive .

Once you have your service id template id. We can set the service email . I done it in This way . If you have better ideas let me know

**Comment by shweta0108** [comment]
Let me know if I can help with this

**Comment by nitin-pandita** [comment]
> Let me know if I can help with this
> 

Need a good design and if you like we can work on it to
